addons/hr_presence/views/hr_employee_views.xml

```diff
@@ -40,12 +40,12 @@
                 <div class="oe_kanban_content">
                     <div class="oe_kanban_row">
                         <field name="hr_presence_state" invisible="1"/>
-                        <button name="action_set_present" type="object" class="btn btn-primary btn-sm mt8 fa fa-check" title="Set as present" attrs="{'invisible': [('hr_presence_state', '=', 'present')]}"> </button>
-                        <button name="action_set_absent" type="object" class="btn btn-primary btn-sm mt8 fa fa-times" title="Set as absent" attrs="{'invisible': [('hr_presence_state', '!=', 'present')]}"> </button>
+                        <button name="action_set_present" type="object" class="btn btn-primary btn-sm mt8 fa fa-check me-1" title="Set as present" attrs="{'invisible': [('hr_presence_state', '=', 'present')]}"> </button>
```

Comment:
why not using `gap-1` on the parent?

Reply:
This was to avoid changing the display property of the parent since it's using a `d-block` and we can't apply `.gap` on top of that.

Reply:
ok... can we `.d-flex.gap-1`?